Updated mediatransportcontrols.md

touseefbsb opened the pull request:
Added an extra code snippet to show that MediaTransportControls work exactly the same way with MediaPlayerElement as well, it will be an easy copy and paste code snippet who wants to get started with MediaPlayerElement and in this example I am promoting the use of IsCompactOverlayButtonVisible="True", so that developers can be easily discover and be aware of this strong feature.

touseefbsb committed:
Updated mediatransportcontrols.md

Added an extra code snippet to show that MediaTransportControls work exactly the same way with MediaPlayerElement as well, it will be an easy copy and paste code snippet who wants to get started with MediaPlayerElement and in this example I am promoting the use of IsCompactOverlayButtonVisible="True", so that developers can be easily discover and be aware of this strong feature.

mikematteson commented:
@jwmsft : PR **passed** acceptance checklist review, but please note the following:
- References to Windows 10 updates should avoid "friendly names" and use only version numbers (e.g., "Windows 10, version *YYMM*").
- Please correct the submitted text for spelling and spacing as needed.

Otherwise, please review for technical accuracy, language, etc., and approve/merge if appropriate. Thanks!

touseefbsb committed:
updated the windows 10 versions names

windows 10 version names in previous commit have been updated as per following format : "Windows 10, version YYMM"

jwmsft committed:
Update mediatransportcontrols.md

jwmsft commented:
@touseefbsb, thanks for the contribution.
